interface/main/tabs/js/tabs_view_model.js

```diff
@@ -324,7 +330,7 @@ function menuActionClick(data,evt)
 
         navigateTab(webroot_url + dataurl, data.target, function () {
             activateTabByName(data.target,true);
-        });
+        },dataLabel);
```

Comment:
to allow the fix for the undefined below (this xl is not impacted by undefined since it is always called from the menu and not during login)
```
},xl("Loading") + " " + dataLabel);
```